.travis/runpylint.sh

```diff
@@ -12,12 +12,24 @@
 
 # The given command line arguments are passed to custom_pylint.py.
 
+# Environment variables:
+#
+#   RUN_PYLINT_SETUP_MODULE_UTILS
+#     if set to an arbitrary non-empty value, the environment will be
```

Comment:
just a remark: Requesting a specific value such as `yes` makes it easier to allow other values in the future, too. For example the coveralls setting allows yes, strict and debug. Therefore it might make sense to decide on a specific value for boolean true here.

Reply:
The problem is that every other use of a boolean valued variable in the scripts assumes `-z "${VAR}"` is `false` and `-n "${VAR}"` or `[[ "${VAR}" ]]` is `true`.  If I make `RUN_PYLINT_SETUP_MODULE_UTILS` and `RUN_PYTEST_SETUP_MODULE_UTILS` have an actual values `no/yes` or `false/true` or something like that in this PR, that will be inconsistent with all other usage of boolean valued variables.
Note that `LSR_PUBLISH_COVERAGE` is not a boolean valued variable, and it doesn't allow `yes` as a value (or at least, once https://github.com/linux-system-roles/network/pull/186 is merged, `yes` will not be a valid value).

Reply:
IIRC, "yes" was the proposed value for `LSR_PUBLISH_COVERAGE` in one of the other discussions. Just to clarify: If you prefer an arbitrary value as "true", I will not block this PR for this.

Reply:
> IIRC, "yes" was the proposed value for `LSR_PUBLISH_COVERAGE` in one of the other discussions.

It may have been, but https://github.com/linux-system-roles/network/pull/186 changes that - with that PR, the valid values are `strict`, `debug`, or `normal` (`normal` replaces `yes` as the value that means "run coverage normally").  And 186 maintains the convention in the test code that the empty value (`-z ${LSR_PUBLISH_COVERAGE}`) means `off` or `false` (as opposed to having a specific `off` or `false` value).

> Just to clarify: If you prefer an arbitrary value as "true", I will not block this PR for this.

I "prefer" it in the sense that, while I recognize that it is not ideal, I don't want to make a change which will be confusing and inconsistent, and I also don't want this PR to include a change to every other util script to change the behavior of every other boolean valued variable.